src/sentry/uptime/models.py

```diff
@@ -363,6 +375,24 @@ def create_detector_from_project_subscription(project_sub: ProjectUptimeSubscrip
         organization=project_sub.project.organization,
         source_id=str(project_sub.uptime_subscription_id),
     )
+    condition_group = DataConditionGroup.objects.create(
+        organization=project_sub.project.organization,
+    )
+    # TODO(epurkhiser): Should we be creating a new data condition + group for
```

Comment:
Open question for this PR

Reply:
We should create a new condition + group for each uptime detector.

That allows us to compose these a bit more; and reuse the `type`s and the `comparison` for different product verticals -- then store all the user configuration as well. 

I could def seeing the `comparison` + `type` with a mapping to data sources or something though 🤔 -- i can have a think on this to see if we can reduce replication.

Reply:
Yeah sounds good. me and @wedamija also realized we might want to add more configurable conditions such as "response contains string" etc. So we wouldn't want to have to split them out later.